### DISCUSSION TITLE
策略模板支持

### DISCUSSION THREAD

**Discussion by RickyLoveFreedom** [wondertrader/wondertrader#208] [想法/Ideas]
![image](https://github.com/wondertrader/wondertrader/assets/21985820/d985fcfc-1e42-4516-a179-e55ea0786701)
    目前很多量化平台(聚宽，ricequant开源的rqalpha,掘金等)都采用的图中类似zipline的策略模板，所以问下wt能否实现图中所示的：定时+事件驱动的策略模板，好用于开发一些涉及多个定时任务的选股策略（每月根据市值初选，每日精选执行等）。
    虽然SEL引擎也能支持定时任务，但是据我了解到的是，定时是在策略外触发的，一个定时任务绑定一个策略，
无法在策略内部自由组合各种定时任务，导致学习和迁移成本还是有点高。

**Comment by Billy1900** [comment]
目前应该是不行的，特别是机器学习这块
